index.js

```diff
@@ -127,8 +152,11 @@ function getFreePort(possiblePort) {
   )
 }
 
-function makeSureTempDirExist(dir) {
+function makeSureTempDirExist(dir, useReplicaSet) {
   try {
+    if (useReplicaSet) {
+      fs.rmdirSync(dir, { recursive: true })
+    }
```

Comment:
This is needed because of [this issue](https://github.com/nodkz/mongodb-memory-server/issues/278) in `mongodb-memory-server`.
Basically we cannot reuse a previously created `tmp` directory between runs.